tests/source_storages/test_source_storages.py

```diff
@@ -10,9 +10,9 @@
     PlainTextDocumentHandler,
     RagnaException,
 )
-from ragna.source_storages import Chroma, LanceDB, RagnaDemoSourceStorage
+from ragna.source_storages import Chroma, LanceDB, QdrantDB, RagnaDemoSourceStorage
 
-SOURCE_STORAGES = [Chroma, LanceDB, RagnaDemoSourceStorage]
+SOURCE_STORAGES = [Chroma, LanceDB, QdrantDB, RagnaDemoSourceStorage]
```

Comment:
The fact that we can plug this in here and all tests "just work" is a little awkward since we are looking into failures for more recent Chroma versions in #523. @Anush008 could you join the discussion there?